### PR TITLE
homework topic

### PR DESCRIPTION
**Pull request by ABC-Dragon-Buster**
written and added a homework/assignment topic for PR, after consulting some of the fellow writers, the final product made use of a choice style (yes/no).

### CODE REVIEW

**Comment by Kiosho** [comment]
I like the application to school and work. So it could apply to both.

m 1a"I hope that could've been of some use to you."

Make sure you put a space between expression and quote. 'm 1a "I hope'

File: Monika After Story/game/script-topics.rpy [hunk 1]
```diff
@@ -4703,6 +4703,33 @@ label monika_write:
...
+           m "I also suggest finishing tasks as early as possible, as cramming them last minute can be quite problematic."
+           m 1a"I hope that could've been of some use to you."
+           m 2g "I don't want my sweetheart to be struggling in school or work, okay?"
+           m 1b "I'm sure you'll be just fine, ahaha~"
+       "No.":
+          m 1a "I'm glad to hear that then."
+          m 1d "Some people have a hard time dealing with work and it starts to pile up a lot."
+          m 1a "So keep up the good work, [player]. Don't stress yourself out too much."
+          m "I believe in you~"
+          return
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: Remove one level of indentation from the `return`. Should look like this:
```
label this_is_label:
    m "bunch of dialogue"
    menu:
        "choice:"
            "more dialogue"
    return
```

**Comment by Winblows2** [comment]
Make sure new topic changes merge into *content*, not master. Master is for small bugfixes & typos.

**Comment by monikLover** [comment]
Closing for now due to the PR basing on the wrong branch. All topics go to "content" not "master."

**Pull request closed without merging**